Base/QTGUI/qSlicerSaveDataDialog.cxx

```diff
@@ -1296,6 +1402,17 @@ void qSlicerSaveDataDialogPrivate::updateOptionsWidget(int row)
   this->FileWidget->resizeColumnToContents(OptionsColumn);
 }
 
+//-----------------------------------------------------------------------------
+void qSlicerSaveDataDialogPrivate
+::updateUserMessagesItem(int row)
+{
+vtkMRMLStorableNode * const node = vtkMRMLStorableNode::SafeDownCast(this->object(row));
```

Comment:
Adding the extra `const` is not something we do consistently ... I suggest to remove it

Reply:
I think of this `const` as a unit test to prevent a regression.  If I try to modify my local variable `node`, I want the compiler to let me know.